channeldb/payments.go

```diff
@@ -597,28 +597,7 @@ func (d *DB) QueryPayments(query PaymentsQuery) (PaymentsResponse, error) {
 		// literally have to traverse the cursor linearly, which can
```

Comment:
typo in commit msg: toyal -> total

Reply:
done 6fecd6211c17aa7fa7e7b7bd4911daab7bbf5f3e